gli.gemspec

```diff
@@ -12,6 +12,15 @@ spec = Gem::Specification.new do |s|
   s.summary = "Build command-suite CLI apps that are awesome."
   s.description = "Build command-suite CLI apps that are awesome.  Bootstrap your app, add commands, options and documentation while maintaining a well-tested idiomatic command-line app"
 
+  s.metadata = {
+    'yard.run'          => 'yard',
```

Comment:
What's this one do?  Is it just "what command do I run to do yard doc generation?"

Reply:
> You can also add the following to your .gemspec to have YARD document your gem
on install:
>
>    spec.metadata["yard.run"] = "yri" # use "yard" to build full HTML docs.

or

https://github.com/lsegal/yard/issues/1187